hooks/pre-command

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -euo pipefail
+
+_SOURCE="${BASH_SOURCE[0]}"
+[ -z "${_SOURCE:-}" ] && _SOURCE="${0}"
+BASEDIR="$( cd "$( dirname "${_SOURCE}" )" && cd .. && pwd )"
```

Comment:
when used as plugin in another build, where will this end up? will the credentials file stick around between builds?

Reply:
Updated to use a temporary directory